packages/ppxx/ppxx.1.3.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+version: "1.3.1"
+authors: "Jun Furuse"
+maintainer: "jun.furuse@gmail.com"
+homepage: "http://bitbucket.org/camlspotter/ppxx"
+bug-reports: "https://bitbucket.org/camlspotter/ppxx/issues?status=new&status=open"
+dev-repo: "hg://https://bitbucket.org/camlspotter/ppxx"
+build: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-build" ]
+]
+install: [
+  [ "ocaml" "setup.ml" "-install" ]
+]
+remove: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-uninstall" ]
+]
+depends: [
+  "ocamlfind" { build }
+  "omake" { build }
+  "ppx_tools" { >= "5.0" }
+]
+available: [
+  ocaml-version = "4.04.0"
```

Comment:
would it be possible to relax this so that it copes with point releases of 4.04 (which are unlikely to change the AST)?